run.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+function print_usage {
+    echo "Usage: $0 [options] command"
+    echo "command: serve | watch | docker | update | build | clean | test | start"
+    echo "   -h      print this help"
+    echo "   -c | --common-path <dir>"
+    echo "           Use this directory for the common libraries. Needs -r."
+    echo "   -r | --regenerate-common"
+    echo "           Fetch the common libraries, either from a local directory"
+    echo "           specified with -c, or from pypi is -c is omitted"
+}
+
+# Read command-line options
+while [[ $# > 1 ]]; do
+    key="$1"
+    case $key in
+        -h|--help)
+        print_usage
+        exit
+        ;;
+        -c|--common-path)
+        COMMONPATH="$2"
+        shift
+        ;;
+        -r|--regenerate-common)
+        REGENERATE=YES
+        ;;
+        *)
+        echo "Unknown option $1"
+        exit
+        ;;
+    esac
+    shift
+done
+
+COMMAND=$1
+
+case "$COMMAND" in
+    clean)
+        git clean -dfX
+        ;;
+    serve | watch | docker | update | build | clean | test | start)
+        APP=cashbook
+        PORT=8001
+        BROWSERSYNC_PORT=3001
+        BROWSERSYNC_UI_PORT=3031
+        VIRTUAL_ENV_DIR=venv
+        NODE_MODULES=node_modules
+        MTP_COMMON_NODE_MODULES=$NODE_MODULES/money-to-prisoners-common
+
+        echo "Creating Python virtual environment"
+        virtualenv --python=python3 $VIRTUAL_ENV_DIR >/dev/null
+        source $VIRTUAL_ENV_DIR/bin/activate >/dev/null
```

Comment:
No, I mean $VIRTUAL_ENV. As in, if there is a virtual env already active, don't try and generate the venv/ dir.